README.md

```diff
@@ -14,88 +26,150 @@ To get started with the Ada Trader project follow these steps:
+
+For this reason advanced traders rely heavily on more advanced order types, primarily the "[limit order](#trading-vocabulary)". A limit order is similar to a market order, with two major exceptions: the limit order includes a "target price", and it remains "open" until cancelled by the trader or the target price is reached. When the current price of the stock reaches the order's target price then the order is executed (creating a new trade), and the order disappears.
+
+### Interface Details
+In the Ada Trader interface the order system can be found in the bottom panel. There are two sections to this panel: the "open orders" list and the order entry form.
+
+The right side of the bottom panel has a form for creating new limit orders. This form has a drop-down list of available stock symbols, a text input for the order's target price, and a Buy and Sell button to create the order.
+
+The left side of the bottom panel has a list of all open orders. Orders are listed with the oldest at the top. Each open order entry in the list includes the symbol being ordered, whether it is a buy or sell order, the target price, and a cancel button.
+
+### User Stories
+As a user, I can:
+  - See a list of open orders in the left side of the bottom panel, including the:
+    - Symbol for each order
+    - Buy/Sell indicator for each order
+    - [Target price](#trading-vocabulary) for each order
+    - Cancel button for each order
+  - See an order entry form in the right side of the bottom panel, including the:
+    - Symbol selection drop-down, which lists all symbols shown in the quote ticker
+    - Target price input box
+    - Buy button
+    - Sell button
+  - Click the Cancel button on each order in the open orders list
+  - Click the Buy button in the order entry form
+  - Click the Sell button in the order entry form
+
+As a user, when I:
+  - Click the Cancel button for an order:
+    - That order is removed from the open orders list
+    - That order will never execute
+  - Click Buy in the order entry form:
+    - If the target price is blank OR is **greater than** the current market price:
+      - That order is not created and (TODO: error display?)
+    - If the target price is **less than or equal** to the current market price:
+      - A new Buy order is added to the bottom of the open orders list, with the:
+        - Symbol from the the form
+        - Target price from the form
+  - Click Sell in the order entry form:
+    - If the target price is blank OR is **less than** the current market price:
+      - That order is not created and (TODO: error display?)
+    - If the target price is **greater than or equal to** the current market price:
+      - A new Sell order is added to the bottom of the open orders list, with the:
+        - Symbol from the the form
+        - Target price from the form
+
+Additionally, when:
+  - The price of a stock quote changes:
```

Comment:
This question is related to the one above about how the simulator is going to work. Based on that i'm interested in your gauge on how easy/difficult it will be to be able to match the price in the order exactly.

Reply:
Right now the simulator only shifts in increments of 10 cents, and at most goes up or down $1.00 per time step (1 second). I hadn't thought about your concern before, but I agree there's a lot of room for people to enter in order prices that don't match exactly.

We could fix that by having them actually implement the more realistic form of limit orders, which is that the market price only needs to match or be better (higher for sells, lower for buys), and then the limit order will execute.

I wasn't sure how much more complicated it would be to explain that so I tried to leave it out, but I think it would result in stuff just not working well if they don't do it that way.

Reply:
I think that the great (or less) logic is more confusing. But I also think that it will be near impossible with the randomization of changes to actually get the limit order to execute. Alternatively, we could adjust the simulator to just change by 50 cents, which would make hitting that number more likely.